example/ios/Flutter/flutter_export_environment.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
+export "FLUTTER_ROOT=/Users/kozw/flutter"
```

Comment:
Note how some of the generated Flutter files are linked to the specifics of the developer work station, like paths.
I usually add such files into the `.gitignore`.